<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>910</width>
    <height>615</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>AudNauseum</string>
  </property>
  <widget class="QDial" name="dial_volume">
   <property name="geometry">
    <rect>
     <x>714</x>
     <y>357</y>
     <width>111</width>
     <height>101</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QDial {
    color: #333;
    border: 2px solid #555;
    border-radius: 20px;
    border-style: outset;
    background: qradialgradient(
        cx: 0.3, cy: -0.4, fx: 0.3, fy: -0.4,
        radius: 1.35, stop: 0 #fff, stop: 1 #111
        );
    padding: 5px;
    }
</string>
   </property>
   <property name="maximum">
    <number>50</number>
   </property>
  </widget>
  <widget class="QLCDNumber" name="lcdNumber">
   <property name="geometry">
    <rect>
     <x>304</x>
     <y>367</y>
     <width>191</width>
     <height>91</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::Box</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <property name="smallDecimalPoint">
    <bool>false</bool>
   </property>
   <property name="digitCount">
    <number>4</number>
   </property>
   <property name="segmentStyle">
    <enum>QLCDNumber::Filled</enum>
   </property>
   <property name="value" stdset="0">
    <double>0.000000000000000</double>
   </property>
   <property name="intValue" stdset="0">
    <number>0</number>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_record">
   <property name="geometry">
    <rect>
     <x>151</x>
     <y>100</y>
     <width>40</width>
     <height>40</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>40</width>
     <height>40</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
    color: #333;
    border: 2px solid #555;
    border-radius: 20px;
    border-style: outset;
    background: qradialgradient(
        cx: 0.3, cy: -0.4, fx: 0.3, fy: -0.4,
        radius: 1.35, stop: 0 #fff, stop: 1 #912
        );
    padding: 5px;
    }

QPushButton:hover {
    background: qradialgradient(
        cx: 0.3, cy: -0.4, fx: 0.3, fy: -0.4,
        radius: 1.35, stop: 0 #fff, stop: 1 #bbb
        );
    }

QPushButton:pressed {
    border-style: inset;
    background: qradialgradient(
        cx: 0.4, cy: -0.1, fx: 0.4, fy: -0.1,
        radius: 1.35, stop: 0 #fff, stop: 1 #ddd
        );
    }</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="checkable">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_play">
   <property name="geometry">
    <rect>
     <x>221</x>
     <y>100</y>
     <width>40</width>
     <height>40</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>40</width>
     <height>40</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
    color: #333;
    border: 2px solid #555;
    border-radius: 20px;
    border-style: outset;
    background: qradialgradient(
        cx: 0.3, cy: -0.4, fx: 0.3, fy: -0.4,
        radius: 1.35, stop: 0 #fff, stop: 1 #192
        );
    padding: 5px;
    }

QPushButton:hover {
    background: qradialgradient(
        cx: 0.3, cy: -0.4, fx: 0.3, fy: -0.4,
        radius: 1.35, stop: 0 #fff, stop: 1 #bbb
        );
    }

QPushButton:pressed {
    border-style: inset;
    background: qradialgradient(
        cx: 0.4, cy: -0.1, fx: 0.4, fy: -0.1,
        radius: 1.35, stop: 0 #fff, stop: 1 #ddd
        );
    }</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="checkable">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_stop">
   <property name="geometry">
    <rect>
     <x>291</x>
     <y>100</y>
     <width>40</width>
     <height>40</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>40</width>
     <height>40</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
    color: #333;
    border: 2px solid #555;
    border-radius: 20px;
    border-style: outset;
    background: qradialgradient(
        cx: 0.3, cy: -0.4, fx: 0.3, fy: -0.4,
        radius: 1.35, stop: 0 #fff, stop: 1 #111
        );
    padding: 5px;
    }

QPushButton:hover {
    background: qradialgradient(
        cx: 0.3, cy: -0.4, fx: 0.3, fy: -0.4,
        radius: 1.35, stop: 0 #fff, stop: 1 #bbb
        );
    }

QPushButton:pressed {
    border-style: inset;
    background: qradialgradient(
        cx: 0.4, cy: -0.1, fx: 0.4, fy: -0.1,
        radius: 1.35, stop: 0 #fff, stop: 1 #ddd
        );
    }</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="checkable">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>157</x>
     <y>150</y>
     <width>31</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>REC</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_7">
   <property name="geometry">
    <rect>
     <x>224</x>
     <y>150</y>
     <width>31</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>PLAY</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_8">
   <property name="geometry">
    <rect>
     <x>293</x>
     <y>150</y>
     <width>41</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>STOP</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_add_track">
   <property name="geometry">
    <rect>
     <x>449</x>
     <y>100</y>
     <width>40</width>
     <height>40</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>40</width>
     <height>40</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
    color: #333;
    border: 2px solid #555;
    border-radius: 20px;
    border-style: outset;
    background: qradialgradient(
        cx: 0.3, cy: -0.4, fx: 0.3, fy: -0.4,
        radius: 1.35, stop: 0 #fff, stop: 1 #111
        );
    padding: 5px;
    }

QPushButton:hover {
    background: qradialgradient(
        cx: 0.3, cy: -0.4, fx: 0.3, fy: -0.4,
        radius: 1.35, stop: 0 #fff, stop: 1 #bbb
        );
    }

QPushButton:pressed {
    border-style: inset;
    background: qradialgradient(
        cx: 0.4, cy: -0.1, fx: 0.4, fy: -0.1,
        radius: 1.35, stop: 0 #fff, stop: 1 #ddd
        );
    }</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="checkable">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_solo_track">
   <property name="geometry">
    <rect>
     <x>450</x>
     <y>280</y>
     <width>40</width>
     <height>40</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>40</width>
     <height>40</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
    color: #333;
    border: 2px solid #555;
    border-radius: 20px;
    border-style: outset;
    background: qradialgradient(
        cx: 0.3, cy: -0.4, fx: 0.3, fy: -0.4,
        radius: 1.35, stop: 0 #fff, stop: 1 #17a
        );
    padding: 5px;
    }

QPushButton:hover {
    background: qradialgradient(
        cx: 0.3, cy: -0.4, fx: 0.3, fy: -0.4,
        radius: 1.35, stop: 0 #fff, stop: 1 #bbb
        );
    }

QPushButton:pressed {
    border-style: inset;
    background: qradialgradient(
        cx: 0.4, cy: -0.1, fx: 0.4, fy: -0.1,
        radius: 1.35, stop: 0 #fff, stop: 1 #ddd
        );
    }</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="checkable">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_new_track">
   <property name="geometry">
    <rect>
     <x>450</x>
     <y>220</y>
     <width>40</width>
     <height>40</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>40</width>
     <height>40</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
    color: #333;
    border: 2px solid #555;
    border-radius: 20px;
    border-style: outset;
    background: qradialgradient(
        cx: 0.3, cy: -0.4, fx: 0.3, fy: -0.4,
        radius: 1.35, stop: 0 #fff, stop: 1 #192
        );
    padding: 5px;
    }

QPushButton:hover {
    background: qradialgradient(
        cx: 0.3, cy: -0.4, fx: 0.3, fy: -0.4,
        radius: 1.35, stop: 0 #fff, stop: 1 #bbb
        );
    }

QPushButton:pressed {
    border-style: inset;
    background: qradialgradient(
        cx: 0.4, cy: -0.1, fx: 0.4, fy: -0.1,
        radius: 1.35, stop: 0 #fff, stop: 1 #ddd
        );
    }</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="checkable">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label_9">
   <property name="geometry">
    <rect>
     <x>510</x>
     <y>293</y>
     <width>41</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>SOLO</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_10">
   <property name="geometry">
    <rect>
     <x>509</x>
     <y>110</y>
     <width>81</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>ADD TRACK</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_11">
   <property name="geometry">
    <rect>
     <x>510</x>
     <y>230</y>
     <width>41</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>NEW</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_12">
   <property name="geometry">
    <rect>
     <x>79</x>
     <y>509</y>
     <width>341</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>48</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>AUDNAUSEUM</string>
   </property>
  </widget>
  <widget class="QSpinBox" name="spinBox_context">
   <property name="geometry">
    <rect>
     <x>534</x>
     <y>366</y>
     <width>141</width>
     <height>91</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>48</pointsize>
     <weight>75</weight>
     <italic>true</italic>
     <bold>true</bold>
    </font>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
   <property name="buttonSymbols">
    <enum>QAbstractSpinBox::PlusMinus</enum>
   </property>
   <property name="minimum">
    <number>1</number>
   </property>
   <property name="maximum">
    <number>20</number>
   </property>
  </widget>
  <widget class="QLabel" name="label_14">
   <property name="geometry">
    <rect>
     <x>744</x>
     <y>463</y>
     <width>61</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>VOLUME</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_15">
   <property name="geometry">
    <rect>
     <x>510</x>
     <y>170</y>
     <width>81</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>REM TRACK</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_rem_track">
   <property name="geometry">
    <rect>
     <x>450</x>
     <y>160</y>
     <width>40</width>
     <height>40</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>40</width>
     <height>40</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
    color: #333;
    border: 2px solid #555;
    border-radius: 20px;
    border-style: outset;
    background: qradialgradient(
        cx: 0.3, cy: -0.4, fx: 0.3, fy: -0.4,
        radius: 1.35, stop: 0 #fff, stop: 1 #111
        );
    padding: 5px;
    }

QPushButton:hover {
    background: qradialgradient(
        cx: 0.3, cy: -0.4, fx: 0.3, fy: -0.4,
        radius: 1.35, stop: 0 #fff, stop: 1 #bbb
        );
    }

QPushButton:pressed {
    border-style: inset;
    background: qradialgradient(
        cx: 0.4, cy: -0.1, fx: 0.4, fy: -0.1,
        radius: 1.35, stop: 0 #fff, stop: 1 #ddd
        );
    }</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="checkable">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="Line" name="line">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>10</y>
     <width>871</width>
     <height>20</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>758</x>
     <y>280</y>
     <width>211</width>
     <height>47</height>
    </rect>
   </property>
   <property name="text">
    <string>REVERSE</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>758</x>
     <y>230</y>
     <width>81</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>SLIP/TEMPO</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>758</x>
     <y>170</y>
     <width>71</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>PITCH/DIV</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_pitch">
   <property name="geometry">
    <rect>
     <x>698</x>
     <y>160</y>
     <width>40</width>
     <height>40</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>40</width>
     <height>40</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
    color: #333;
    border: 2px solid #555;
    border-radius: 20px;
    border-style: outset;
    background: qradialgradient(
        cx: 0.3, cy: -0.4, fx: 0.3, fy: -0.4,
        radius: 1.35, stop: 0 #fff, stop: 1 #111
        );
    padding: 5px;
    }

QPushButton:hover {
    background: qradialgradient(
        cx: 0.3, cy: -0.4, fx: 0.3, fy: -0.4,
        radius: 1.35, stop: 0 #fff, stop: 1 #bbb
        );
    }

QPushButton:pressed {
    border-style: inset;
    background: qradialgradient(
        cx: 0.4, cy: -0.1, fx: 0.4, fy: -0.1,
        radius: 1.35, stop: 0 #fff, stop: 1 #ddd
        );
    }</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="checkable">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_slip">
   <property name="geometry">
    <rect>
     <x>698</x>
     <y>220</y>
     <width>40</width>
     <height>40</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>40</width>
     <height>40</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
    color: #333;
    border: 2px solid #555;
    border-radius: 20px;
    border-style: outset;
    background: qradialgradient(
        cx: 0.3, cy: -0.4, fx: 0.3, fy: -0.4,
        radius: 1.35, stop: 0 #fff, stop: 1 #111
        );
    padding: 5px;
    }

QPushButton:hover {
    background: qradialgradient(
        cx: 0.3, cy: -0.4, fx: 0.3, fy: -0.4,
        radius: 1.35, stop: 0 #fff, stop: 1 #bbb
        );
    }

QPushButton:pressed {
    border-style: inset;
    background: qradialgradient(
        cx: 0.4, cy: -0.1, fx: 0.4, fy: -0.1,
        radius: 1.35, stop: 0 #fff, stop: 1 #ddd
        );
    }</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="checkable">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_reverse">
   <property name="geometry">
    <rect>
     <x>698</x>
     <y>280</y>
     <width>40</width>
     <height>40</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>40</width>
     <height>40</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
    color: #333;
    border: 2px solid #555;
    border-radius: 20px;
    border-style: outset;
    background: qradialgradient(
        cx: 0.3, cy: -0.4, fx: 0.3, fy: -0.4,
        radius: 1.35, stop: 0 #fff, stop: 1 #111
        );
    padding: 5px;
    }

QPushButton:hover {
    background: qradialgradient(
        cx: 0.3, cy: -0.4, fx: 0.3, fy: -0.4,
        radius: 1.35, stop: 0 #fff, stop: 1 #bbb
        );
    }

QPushButton:pressed {
    border-style: inset;
    background: qradialgradient(
        cx: 0.4, cy: -0.1, fx: 0.4, fy: -0.1,
        radius: 1.35, stop: 0 #fff, stop: 1 #ddd
        );
    }</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="checkable">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_pan_beats">
   <property name="geometry">
    <rect>
     <x>698</x>
     <y>100</y>
     <width>40</width>
     <height>40</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>40</width>
     <height>40</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
    color: #333;
    border: 2px solid #555;
    border-radius: 20px;
    border-style: outset;
    background: qradialgradient(
        cx: 0.3, cy: -0.4, fx: 0.3, fy: -0.4,
        radius: 1.35, stop: 0 #fff, stop: 1 #111
        );
    padding: 5px;
    }

QPushButton:hover {
    background: qradialgradient(
        cx: 0.3, cy: -0.4, fx: 0.3, fy: -0.4,
        radius: 1.35, stop: 0 #fff, stop: 1 #bbb
        );
    }

QPushButton:pressed {
    border-style: inset;
    background: qradialgradient(
        cx: 0.4, cy: -0.1, fx: 0.4, fy: -0.1,
        radius: 1.35, stop: 0 #fff, stop: 1 #ddd
        );
    }</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="checkable">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label_16">
   <property name="geometry">
    <rect>
     <x>758</x>
     <y>110</y>
     <width>71</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>PAN/BEATS</string>
   </property>
  </widget>
  <widget class="Line" name="line_2">
   <property name="geometry">
    <rect>
     <x>68</x>
     <y>80</y>
     <width>271</width>
     <height>20</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>128</x>
     <y>70</y>
     <width>161</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>TRANSPORT CONTROLS</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>724</x>
     <y>70</y>
     <width>91</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>EFFECTS (FX)</string>
   </property>
  </widget>
  <widget class="Line" name="line_3">
   <property name="geometry">
    <rect>
     <x>690</x>
     <y>80</y>
     <width>151</width>
     <height>20</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="Line" name="line_4">
   <property name="geometry">
    <rect>
     <x>446</x>
     <y>80</y>
     <width>141</width>
     <height>20</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QLabel" name="label_17">
   <property name="geometry">
    <rect>
     <x>460</x>
     <y>70</y>
     <width>121</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>TRACK CONTROLS</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_metro_indicator">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>134</x>
     <y>397</y>
     <width>40</width>
     <height>40</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>40</width>
     <height>40</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
    color: #333;
    border: 2px solid #555;
    border-radius: 20px;
    border-style: outset;
    background: qradialgradient(
        cx: 0.3, cy: -0.4, fx: 0.3, fy: -0.4,
        radius: 1.35, stop: 0 #fff, stop: 1 #ddd
        );
    padding: 5px;
    }

QPushButton:hover {
    background: qradialgradient(
        cx: 0.3, cy: -0.4, fx: 0.3, fy: -0.4,
        radius: 1.35, stop: 0 #fff, stop: 1 #bbb
        );
    }

QPushButton:pressed {
    border-style: inset;
    background: qradialgradient(
        cx: 0.4, cy: -0.1, fx: 0.4, fy: -0.1,
        radius: 1.35, stop: 0 #fff, stop: 1 #ddd
        );
    }</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_19">
   <property name="geometry">
    <rect>
     <x>199</x>
     <y>443</y>
     <width>51</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>ON/OFF</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_metro_on_off">
   <property name="geometry">
    <rect>
     <x>204</x>
     <y>397</y>
     <width>40</width>
     <height>40</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>40</width>
     <height>40</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
    color: #333;
    border: 2px solid #555;
    border-radius: 20px;
    border-style: outset;
    background: qradialgradient(
        cx: 0.3, cy: -0.4, fx: 0.3, fy: -0.4,
        radius: 1.35, stop: 0 #fff, stop: 1 #111
        );
    padding: 5px;
    }

QPushButton:hover {
    background: qradialgradient(
        cx: 0.3, cy: -0.4, fx: 0.3, fy: -0.4,
        radius: 1.35, stop: 0 #fff, stop: 1 #bbb
        );
    }

QPushButton:pressed {
    border-style: inset;
    background: qradialgradient(
        cx: 0.4, cy: -0.1, fx: 0.4, fy: -0.1,
        radius: 1.35, stop: 0 #fff, stop: 1 #ddd
        );
    }</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="checkable">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="Line" name="line_5">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>377</y>
     <width>131</width>
     <height>20</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QLabel" name="label_20">
   <property name="geometry">
    <rect>
     <x>144</x>
     <y>367</y>
     <width>91</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>METRONOME</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_21">
   <property name="geometry">
    <rect>
     <x>544</x>
     <y>467</y>
     <width>131</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>SELECTED TRACK</string>
   </property>
  </widget>
  <widget class="Line" name="line_6">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>590</y>
     <width>871</width>
     <height>20</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="Line" name="line_7">
   <property name="geometry">
    <rect>
     <x>12</x>
     <y>20</y>
     <width>20</width>
     <height>581</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Vertical</enum>
   </property>
  </widget>
  <widget class="Line" name="line_8">
   <property name="geometry">
    <rect>
     <x>881</x>
     <y>20</y>
     <width>20</width>
     <height>581</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Vertical</enum>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_rec_indicator">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>100</y>
     <width>40</width>
     <height>40</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>40</width>
     <height>40</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
    color: #333;
    border: 2px solid #555;
    border-radius: 20px;
    border-style: outset;
    background: qradialgradient(
        cx: 0.3, cy: -0.4, fx: 0.3, fy: -0.4,
        radius: 1.35, stop: 0 #fff, stop: 1 #ddd
        );
    padding: 5px;
    }

QPushButton:hover {
    background: qradialgradient(
        cx: 0.3, cy: -0.4, fx: 0.3, fy: -0.4,
        radius: 1.35, stop: 0 #fff, stop: 1 #bbb
        );
    }

QPushButton:pressed {
    border-style: inset;
    background: qradialgradient(
        cx: 0.4, cy: -0.1, fx: 0.4, fy: -0.1,
        radius: 1.35, stop: 0 #fff, stop: 1 #ddd
        );
    }</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_13">
   <property name="geometry">
    <rect>
     <x>75</x>
     <y>150</y>
     <width>61</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>STATUS</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_18">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>443</y>
     <width>61</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>STATUS</string>
   </property>
  </widget>
  <widget class="QListView" name="listView">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>180</y>
     <width>251</width>
     <height>161</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
